templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -38,7 +38,7 @@ contents:
       if [ -f "$ASSET_DIR/backup/etcd-ca-bundle.crt" ] && [ -f "$ASSET_DIR/backup/etcd-client.crt" ] && [ -f "$ASSET_DIR/backup/etcd-client.key" ]; then
          echo "etcd client certs already backed up and available $ASSET_DIR/backup/"
       else
-        STATIC_DIRS=($(ls -td "${CONFIG_FILE_DIR}"/static-pod-resources/kube-apiserver-pod-[0-9]*))
+        STATIC_DIRS=($(ls -td "${CONFIG_FILE_DIR}"/static-pod-resources/kube-apiserver-pod-[0-9]*)) || true
         if [ "$?" -ne 0 ]; then
```

Comment:
This won't work as expected now in a failure case.

```
#!/bin/bash
  
  set -o errexit
  set -o pipefail
  TEST=($(ls -td ./does/not/exist )) || true
  echo "return is $?"
  return is 0
```
Maybe you want to just wrap this together, one idea would be something like.

```
STATIC_DIRS=($(ls -td "${CONFIG_FILE_DIR}"/static-pod-resources/kube-apiserver-pod-[0-9]*)) || {
  echo "error finding static-pod-resources"
  exit 1
}}
```

Reply:
Thanks. Fixed it.